src/store/todos.test.js

```diff
@@ -25,38 +38,14 @@ describe(`todos reducer`, () => {
         id: 0
       }
     ]}
-    expect(reducer(previousState, todoAdded({description: 'Use Redux', completed: false, id: 1})).todos).toEqual([
-      {
-        description: 'Run the tests',
-        completed: true,
-        id: 0
-      },
-      {
-        description: 'Use Redux',
-        completed: false,
-        id: 1
-      }
-    ])
+    expect(reducer(previousState, todoAdded({description: 'Use Redux', completed: false, id: 1})).todos).toEqual(initialState.todos)
```

Comment:
Codacy has a fix for the issue: [Replace `reducer(previousState,·todoAdded({description:·'Use·Redux',·completed:·false,·id:·1})).todos).toEqual(initialState.todos)` with `⏎······reducer(⏎········previousState,⏎········todoAdded({·description:·"Use·Redux",·completed:·false,·id:·1·})⏎······).todos⏎····).toEqual(initialState.todos);`](https://app.codacy.com/gh/JThurston4/tdd-todo/pullRequest?prid=9315756)
```suggestion
    expect(
```